bash/compile_python_packages.sh

```diff
@@ -2,7 +2,7 @@
 #
 # Dev script to compile python packages from a requirements.in file to a requirements.txt file.
 set -e
-path_to_requirements="${1:-"."}"
+path_to_requirements="python"
```

Comment:
Figured we don't really need ability to run this from multiple places and nicer to just get this consistently, I was tired of changes between "./python/requirements.in" and "python/requirements.in" in `requirements.txt`